### DISCUSSION TITLE
Get w/ host working?

### DISCUSSION THREAD

**Discussion by rodhash** [rest-nvim/rest.nvim#336] [Q&A]
After moving to the new version using luarocks, got some issues using GET w/ host.

Simple GET is working just fine - `Rest run` on line 3
<img width="1530" alt="image" src="https://github.com/rest-nvim/rest.nvim/assets/29671981/890411c0-d695-4c8f-b8ec-7726aee69fd1">

But GET w/ host is throwing some errors:
<img width="1904" alt="image" src="https://github.com/rest-nvim/rest.nvim/assets/29671981/5cd104d9-a882-4536-b7b2-9d0094c5ee6e">

I also have all the required Treesitter parsers installed according to the README.

Am I missing anything? Perhaps something related to the newer version?

Thanks in advance

**Comment by lyohar** [comment]
I have similar issue.

**Reply by lyohar** [reply]
looks like the issue reproduces in Neovim 0.9 when installing via lazy.nvim
I have rolled back to v1.2.1 (still on Neovim 0.9 and lazy)

**Comment by boltlessengineer** [comment]
Closing discussion due to v3 release of both rest.nvim and tree-sitter-http parser.
This should be resolved now. Please open a new issue/discussion if you found something still not working.
